docs/installation_configuration_guide/DesignerConfiguration.md

```diff
@@ -165,6 +165,11 @@ users: [
     identity: "admin"
     roles: ["Admin"]
   },
+  {
+    identity: "234a9sa98234@clientId",
```

Comment:
Do you added by mistake, or intentionally? If intentionally, I think we need to add some notice what this example is trying to tell users in this place. Without it, it looks magical. Why should I configure M2M tech account?